lib/adapters/class-vip-enterprise-search.php

```diff
@@ -25,19 +117,44 @@ class VIP_Enterprise_Search extends Adapter {
 	 * @return array New request arguments.
 	 */
 	public function filter_ep_query_request_args( $request_args, $path, $index, $type ): array {
-		// Try to convert the request body to an array so we can work with it.
+		// Try to convert the request body to an array, so we can work with it.
 		$dsl = json_decode( $request_args['body'], true );
 		if ( ! is_array( $dsl ) ) {
 			return $request_args;
 		}
 
 		// Add our aggregations.
+		if ( $this->get_aggregate_post_dates() ) {
+			$post_date_facet = new Post_Date();
```

Comment:
We're going back and forth here between calling them "aggregations" and "facets" - I'd recommend that we use "aggregations" because that's what Elasticsearch calls them.